### PR TITLE
Fixing compress errors

### PR DESCRIPTION
**Pull request by bruskiza**

### CODE REVIEW

**Comment by bruskiza** [comment]
@miltontony @hodgestar tiny review again please.

File: sideloader/postinstall.sh [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
     npm install -g less
 fi
 
 $pip install -r pip-freeze.txt
-python manage.py migrate --noinput
-python manage.py collectstatic --noinput
+
+$manage migrate --noinput
+$manage collectstatic --noinput
+$manage compress --noinput
```
Comment (miltontony) [inline sideloader/postinstall.sh]: `no such option: --noinput`

**Comment by bruskiza** [comment]
@miltontony ready for re-review.

**Comment by miltontony** [comment]
:+1:

**Pull request merged**